circleci/catapult-publish

```diff
@@ -43,6 +45,33 @@ REPO=$CIRCLE_PROJECT_REPONAME
 SHORT_SHA=${CIRCLE_SHA1:0:7}
 FULL_SHA=${CIRCLE_SHA1}
 
+./install_goci
+
+# Run the validate step using goci and capture the output
+VALIDATE_OUTPUT=$(goci artifact-build-publish-deploy)
+VALIDATE_EXIT_CODE=$?
+
+# In the case of an error - we print the error and will make a PR comment with the error message
+if [[ $VALIDATE_EXIT_CODE -ne 0 ]]; then
```

Comment:
So this script has `-set e` at the top, which means that if any command exits non-zero then the entire script will, so this line is unreachable. 


I think it is probably ok to not make a comment for non-zero exit codes. Since CI will have failed, devs will have to check the CI output anyways.